web-console/knikubevirt/list-view-pf4/list-view-pf4.md

```diff
@@ -0,0 +1,89 @@
+# List View in PF4
+
+This PR describes the changes made in list view in the Virtualization Console when applying them into PF4.
+
+## VM List
+
+The VM list contains columns for the following:
+
+* Name
+* Namespace
+* Status
+  * Running, Off, Error (with custom messages), Cloning, Migrating, Importing, Warning, Pending (with changes that will apply after restart), Unknown
+* Created
+  * Date formatting should follow OpenShift’s console, viewable here and here
+  * If possible, a tooltip should appear on hover with the exact creation date, including year.
+* Node
+* IP Address
+
+### Max number of columns and prioritization
+
+We set 6 columns to be the max number we will display in list views. We also determined priorities/usefulness of each column and set the priority order from left to right.
+
+![VM list six columns](img/vmlist.png)
+
+### Status Popovers and Icons
+
+Statuses should be colored blue to indicate that they can be hovered and/or clicked.
+Hovering over a blue status should display a status popovers with a brief description of the issue or underlying process. Clicking on a ‘Learn more’ link should bring the user to the VM’s Events page for more details.
```

Comment:
Should be click/tap. Hovering should only underline the blue status text.

Another note - “Learn more” with the external link icon (seen below in the Guest Agent example) should link to external documentation, not the VM’s Events page. “View details” without the external link icon would be more appropriate if the status is an error, for example, and clicking would bring the user to the Events page.

Rather than explaining the Status Popover pattern in this documentation section, maybe we should include a collection of what various generic status popovers should look like? One for Error, one for Creating, one for Cloning, etc.? Or maybe those would be included throughout other documentation pages, in which case this section isn’t needed.

Reply:
+andybraren thanks for your comments. 
As for the comment about the status popovers, I added a generic explanation (from the PF documentation) that explains how popovers should look like.